micronaut-maven-plugin/src/main/java/io/micronaut/maven/AbstractDockerMojo.java

```diff
@@ -136,6 +137,14 @@ protected String graalVmVersion() {
         return mavenProject.getProperties().getProperty("graal.version");
     }
 
+    /**
+     * @return the oracle Linux version to use for the GraalVM base image. Defaults to {@value #ORACLE_LINUX_VERSION}
+     */
+    protected String oracleLinuxVersion() {
+        String version = mavenProject.getProperties().getProperty("graal.oracle.linux.version");
```

Comment:
This is not the best way to handle it. Instead, define an attribute like:

```java
@Parameter(property = "micronaut.native-image.ol.version", defaultValue = DEFAULT_ORACLE_LINUX_VERSION)
protected String oracleLinuxVersion;
```

With this, you get for free:
- No need to handle manually the default value case.
- Automatically displaying the javadoc of the attribute in the mojo documentation web page.
- Users get the ability to set the property in the plugin configuration block.